src/content/docs/ai-monitoring/explore-ai-data/view-model-data.mdx

```diff
@@ -0,0 +1,171 @@
+1. Choose time parameters. This tool is flexible: you can make comparisons across different time periods, which lets you see how performance or cost changed before and after a deployment.  
+
+### Compare model performance [#compare-performance]
+
+<img
+    title="Model comparison page: Performance"
+    alt="A screenshot of model comparison"
+    src={aiModelComparisonPerformance}
+/>
+
+<figcaption>
+    Go to **<DoNotTranslate>[one.newrelic.com](https://one.newrelic.com) > All Capabilities > AI monitoring > Model comparison</DoNotTranslate>**: Compare performance between different AI models in your stack.
+</figcaption>
+
+Compare how one model performs within an app against the average performance across all services.
+
+* Conduct A/B tests when testing different prompts during prompt engineering.
+* Compare a model's performance and accuracy during one time window against another time window. From there, determine how a change made to your models affected performance over time. 
+* Evaluate how a model performed during a specific time window when customer traffic peaked.
+
+### Compare model cost [#compare-cost]
+
+<img
+    title="Model comparison page: Cost"
+    alt="A screenshot of model comparison"
+    src={aiModelComparisonCost}
+/>
+
+<figcaption>
+    Go to **<DoNotTranslate>[one.newrelic.com](https://one.newrelic.com) > All Capabilities > AI monitoring > Model comparison</DoNotTranslate>**: Compare cost between different AI models in your stack.
+</figcaption>
+
+The model cost column breaks down completion events into two parts: the prompt given to the model and the final response the model delivers to the end user.
+
+* <DoNotTranslate>**Tokens per completion**</DoNotTranslate>: The token average for all completion events.
+* <DoNotTranslate>**Prompt tokens**</DoNotTranslate>: The token average for prompts. This token average includes prompts created by prompt engineers and end users.
+* <DoNotTranslate>**Completion tokens**</DoNotTranslate>: The number of tokens consumed by the model when it generates the response delivered to the end user.
+
+When analyzing this column, the value for completion tokens and prompt tokens should equal the value in tokens per completion.
+
+## What's next? [#whats-next]
+
+Now that you know how to find your data, you can explore other features that AI monitoring has to offer.
+
+* Want to analyze the performance of your AI app? Check out our doc about [the AI app response pages](/docs/ai-monitoring/explore-ai-data/view-ai-responses).
+* Concerned about sensitive information? [Learn to set up drop filters](/docs/ai-monitoring/drop-sensitive-data).
+* If you want forward user feedback information about your app's AI responses to New Relic, [follow our instructions to update your app code to get user feedback in the UI](/docs/ai-monitoring/customize-agent-ai-monitoring).
```

Comment:
I'd suggest replacing this:
[follow our instructions to update your app code to get user feedback in the UI](/docs/ai-monitoring/customize-agent-ai-monitoring).
with this:
follow our instructions to [update your app code to get user feedback in the UI](/docs/ai-monitoring/customize-agent-ai-monitoring/#enable-feedback).